src/Cmf/MainBundle/Resources/data/posts/2015-02-01.yml

```diff
@@ -0,0 +1,94 @@
+name: "cmf-news-january-2015"
+parent: "/news"
+title: "CMF News Januray 2015"
+label: false
+format: markdown
+template: CmfMainBundle:Cms:news_detail.html.twig
+publish_start_date: "2015-02-01"
+body: |
+    Its quite an exciting time for the CMF, there is talk about the next
+    generation of Jackalope, the Resource stack, rewriting the
+    TreeBrowserBundle and true multi-site support.
+
+    We also had a CMF hackday in December at the Mayflower offices in Würzburg, Germany and we are
+    hoping to have another one in the coming months, if you are interested make your self known
```

Comment:
What do you think about splitting this in more than one sentence? "one in the coming months. If you are interested, make yourself known [...]"